### ISSUE TITLE
Setup and Test Kedro-Viz in VSCode

### ISSUE THREAD

**Issue by jitu5** [kedro-org/vscode-kedro#120]
### Requirements:

- VS Code 1.64.0 or greater
- Python extension for VS Code
- Kedro Project >= 0.19

### Pre-requisite:

- The extension requires `bootstrap_project` in Kedro. You need to ensure that you can run `kedro run` without encountering any immediate errors; otherwise, you may experience a server panic error.
- Select the correct Python interpreter that you use to run the Kedro project by pressing `Shift + Cmd + P`, typing “Python: Select Interpreter,” and hitting Enter.

### Setup:

- You can install pre-release version of extension from marketplace directly
- <img width="585" alt="369378934-39c8594e-6265-4538-9ffa-5eea711d629011111" src="https://github.com/user-attachments/assets/8ea1742a-e0ca-42f4-a132-8b97e6d20e37">
- Restart the extension or VSCode window by pressing `Shift + Cmd + P`, typing ‘Developer: Reload Window,’ and hitting Enter.
- If you are using the Kedro extension for the first time, note that it will only activate when you open a `.yml` or `.py` file in the editor.

### Check:

- To check if the Kedro extension is running properly, navigate to `Output` and select `Kedro` from the drop-down menu.
- It may provide hints and report errors. If errors appear, press `Cmd + Shift + P` to open the VSCode command palette and search for Kedro: Restart Server.
- If there are no errors, open a `pipeline.py` file from your Kedro project and hover over any input or output values to see their details.
- <img width="600" alt="Screenshot 2024-09-20 at 10 01 16 a m" src="https://github.com/user-attachments/assets/4b286386-8202-4e75-a488-87f626a4c3ec">

### Visualize:

- To visualize the Kedro pipeline, press `Shift + Cmd + P`, type ‘Kedro: Run Kedro Viz,’ and hit Enter.

### Test Case 1:

- If you do not have Kedro-Viz `10.0.0` installed in your virtual environment, you will be prompted to install it.
- <img width="250" alt="Screenshot 2024-09-20 at 11 13 31 a m" src="https://github.com/user-attachments/assets/e8d67a13-e194-4ab8-8222-a35c111d59c5">
- If you click on “Install,” you will see a loader notification while the installation is in progress.
- <img width="360" alt="Screenshot 2024-09-20 at 11 13 47 a m" src="https://github.com/user-attachments/assets/a7243b29-8da5-4730-9e02-a6cb67aaa751">
- Once installation is complete, the Kedro-Viz flowchart will appear.
- If you click “Cancel,” a notification will appear, and the operation will be canceled.
- <img width="360" alt="Screenshot 2024-09-20 at 11 17 44 a m" src="https://github.com/user-attachments/assets/036cd3d5-fcd4-41f7-8135-d1865950f33c">

### Test Case 2:

- Once the flowchart loads successfully, click on any data node to navigate to the corresponding `.yml `file.

### Test Case 3:

- Once the flowchart loads successfully, click on any function node to navigate to the corresponding `.py` file.

### Test Case 4:

- Make some changes to your Kedro project that can be visualised in the flowchart (after changes, ensure that you can run `kedro run` without any immediate errors).
- To reflect your changes in the flowchart, press `Cmd + Shift + P`, search for Kedro: Restart Server, and hit Enter.

### Test Case 5:

- Please repeat above steps for other Kedro project.

**Comment by rashidakanchwala** [comment]
I tried the above. All scenarios work for me. Thanks!

**Comment by astrojuanlu** [comment]
This was already done!
